src/effects/backends/builtin/pitchshifteffect.cpp

```diff
@@ -61,12 +69,34 @@ EffectManifestPointer PitchShiftEffect::getManifest() {
     pitch->setNeutralPointOnScale(0.0);
     pitch->setRange(-1.0, 0.0, 1.0);
 
+    EffectManifestParameterPointer semitonesMode = pManifest->addParameter();
+    semitonesMode->setId("semitonesMode");
+    semitonesMode->setName(QObject::tr("Semitones"));
+    semitonesMode->setShortName(QObject::tr("Semitones"));
+    semitonesMode->setDescription(QObject::tr(
+            "Change pitch in semitone-steps instead of smoothly."));
+    semitonesMode->setValueScaler(EffectManifestParameter::ValueScaler::Toggle);
+    semitonesMode->setUnitsHint(EffectManifestParameter::UnitsHint::Unknown);
+    semitonesMode->setRange(0, 1, 1);
+
+    EffectManifestParameterPointer formant = pManifest->addParameter();
+    formant->setId("formant");
+    formant->setName(QObject::tr("Formant"));
+    formant->setShortName(QObject::tr("Formant"));
+    formant->setDescription(QObject::tr(
+            "Preserve formants."));
```

Comment:
```suggestion
            "Preserve formants, the resonance frequencies of human vocal tract and other instruments."));
```

I must admit I was not clear what exactly formats are when starting play with pitch shifting. 
Maybe this one liner is enough to give at least a hint, for others like me. 

Or we may pick a sentence that is more catchy but less exact. 

"Preserve formants to compensate chipmunk or growling voices."

Reply:
```suggestion
            "Preserve the resonant frequencies of human vocal tract and other instruments (formants)"));
```
Thats better imo. Easy explanation upfront and then the technical term afterwards for those that are familiar with it.

Reply:
Pick what works for you, or an entire different string. As long we explain the user a bit more what this is about it should be fine.